operator/config/channels/namespaced-per-namespace-components/kustomization.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ../release-bundle/install-bundle-namespaced/per-namespace-components.yaml
```

Comment:
Ideally we want to put the single file under this directory, so that any changes to the `per-namespace-components` file can be aware of and better traced

Reply:
I think that would be great, but I think we should do that separately!